app-emulation/libvirt/libvirt-5.6.0.ebuild

```diff
@@ -0,0 +1,373 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit autotools bash-completion-r1 eutils linux-info python-any-r1 readme.gentoo-r1 systemd
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://libvirt.org/git/libvirt.git"
+	SRC_URI=""
+	KEYWORDS="amd64 x86"
+	SLOT="0"
+else
+	SRC_URI="https://libvirt.org/sources/${P}.tar.xz"
+	KEYWORDS="amd64 ~arm64 x86"
```

Comment:
When bumping to a new version you have to drop the keywords to unstable, i.e.
`KEYWORDS="~amd64 ~arm64 ~x86"`